proto/content_analysis/sdk/analysis.proto

```diff
@@ -60,12 +60,23 @@ message ContentData {
   optional string file_path = 2;
 }
 
-// Analysis request sent from Google Chrome to agent.
+// Analysis request sent from chrome to backend.
+// The proto in the Chromium codebase is the source of truth, the version here
+// should always be in sync with it (https://osscs.corp.google.com/chromium/chromium/src/+/main:components/enterprise/common/proto/connectors.proto;l=87;drc=a8fb6888aff535f27654f03cd1643868ba066de9).
 message ContentAnalysisRequest {
-  // Token used to correlate requests and responses.
+  // The TokenID for Enterprise-enrolled devices.  This identifies a specific
+  // chrome instance.
+  optional string device_token = 1;
+
+  // Firebase Cloud Messaging token used to notify this client of the verdict.
+  // This identifies a specific chrome instance.
+  optional string fcm_notification_token = 2;
```

Comment:
Sorry my bad, I should have been clearer.  These fields (1 and 2) are used only with cloud content analysis, so let's leave the as reserved.